omod/src/main/java/org/openmrs/module/owa/activator/OwaActivator.java

```diff
@@ -163,4 +170,34 @@ public void stopped() {
 		log.info("OWA Module stopped");
 	}
 	
+	/**
+	 * Used to set the login Url Global Property that directs to the Add-on OWA UI if no UI module
+	 * is installed. Searches against the legacy and Reference Application module on installed UI
+	 * modules.
+	 */
+	private void configureLoginUrl(AppManager appManager) {
+		if (!uiModuleInstalled() && appManager.exists(ADD_ON_MANAGER_APP)) {
```

Comment:
Why don't you just return for the opposite of the above if statement?

Reply:
Ahh, yes. Wondering why it took me so long to capture this. Thanks @dkayiwa :-)